rendering/fbx.ksy

```diff
@@ -0,0 +1,148 @@
+meta:
+  id: fbx
+  title: FBX 3D model interchange format
+  file-extension: fbx
+  encoding: UTF-8
+  endian: le
+  license: MIT
+
+seq:
+  - id: header
+    type: header
+  - id: records
+    type: node_record
+    repeat: until
+    repeat-until: _.end_offset == 0
+
+types:
+
+  header:
+    seq:
+      - id: magic
+        contents: "Kaydara FBX Binary  \0"
+        size: 20
+      - id: more_magic
+        contents: [0x1A, 0x00]
```

Comment:
Looks like `u2`. You can use `valid` here too.

Reply:
Is valid just the same as contents but you get a different exception when it's wrong?

Reply:
`valid` allows multiple ways of validation, `valid: something` is just a shortcut for `valid: eq: something`

Reply:
Ah nice, a declarative predicate language.